products/bash/1-click-install.sh

```diff
@@ -330,4 +330,16 @@ fi
 
 divider
 
+echo "release-assets"
+
+if ! $CURRENT_DIR/release-assetrepo.sh -n "$JOB_NAMESPACE" -r "$assetRepoReplicaCount" -s "$DEFAULT_BLOCK_STORAGE" ; then
```

Comment:
I've just looked over the full 1-click file we don't actually specify a assetRepoReplicaCount as part of the script I think what might be better is that we change the navReplicaCount to replicaCount and then that param can be used for both nav and asset repo